### ISSUE TITLE
go-build-tools repo request

### ISSUE THREAD

**Issue by Aneurysm9** [open-telemetry/community#782]
We have a need for a repository to house build tools for use by the Go API/SDK, the collector, and their associated contrib repositories.  The @open-telemetry/go-maintainers and @open-telemetry/collector-maintainers groups should have merge and approval access for this repo.

**Comment by alolita** [comment]
Can you please provide approval access to @open-telemetry/wg-prometheus too? Ty!

**Comment by alolita** [comment]
@SergeyKanzhelev can you help create this repo? We're blocked on this project - ty!

**Comment by SergeyKanzhelev** [comment]
created: https://github.com/open-telemetry/opentelemetry-go-build-tools

@bogdandrutu @lizthegrey please enable EasyCLA

**Comment by SergeyKanzhelev** [comment]
> Can you please provide approval access to @open-telemetry/wg-prometheus too? Ty!

I'm not sure what the relationship between being in wg and having approval powers. I'd let two maintainers teams mentioned to make a decision

**Comment by Oberon00** [comment]
I hope this repository just created because you don't want to rename `semconvgen` to `go-gen-semconv` in the README?  https://github.com/open-telemetry/build-tools/pull/49#discussion_r661212564

**Comment by lizthegrey** [comment]
CLA configured.

**Comment by alolita** [comment]
Thanks @SergeyKanzhelev appreciate it!

**Comment by SergeyKanzhelev** [comment]
> I hope this repository has not just been created because you don't want to rename `semconvgen` to `go-gen-semconv` in the README? [open-telemetry/build-tools#49 (comment)](https://github.com/open-telemetry/build-tools/pull/49#discussion_r661212564) (I know there was some other disagreement on this PR but the mentioned comment was the only blocking one -- at least from me)

@Aneurysm9 please clarify. I assumed creation of this repo was discussed with the corresponding maintainers. If the preference is to merge repositories, lmk and I can delete newly created repository.

**Comment by Aneurysm9** [comment]
This was discussed with the relevant maintainers.  @Oberon00 is not one of the relevant maintainers.  This approach was agreed among the Go maintainers after weeks of incomprehensible obstruction to our use of a directory in the `build-tools` repo and waiting for spec-approvers to review PRs wholly unrelated to the spec.  That repo is obviously not a good fit for our needs and so we now have a repository that we can use in a way that works for us.

**Comment by SergeyKanzhelev** [comment]
I'm fine either way as long as it works for everybody. @Oberon00 lmk if you have concerns

**Comment by Oberon00** [comment]
I think the problem is that there are not enough approvers for the build tools repository. I think creating a separate folder and CODEOWNERS entry for build-tools (in a separate PR) would be an alternative solution, but I don't see a problem with a separate repository either.
